gr-azure-software-radio/grc/azure_software_radio_rest_api.block.yml

```diff
@@ -0,0 +1,38 @@
+id: rest_api
+label: Rest API
+category: '[Azure software radio]'
+
+parameters:
+-   id: value
+    dtype: string
+    default: '0'
+    hide: all
+-   id: port
+    dtype: int
+    default: 8000
```

Comment:
Could we add in the ability to set a list of readable params, and a list of writable params, just to let users decide what parameters should be exposed, and in what way?